.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,52 @@
+45
+
+WebAssembly
+Wasm
+validator
+ERC
+fuzzer
+SHA
+BLAKE2
+KECCAK
+ABI
+FFI
+UI
+crypto
+deploy
+layed
+dispatchable
+SRML
+encodable
+decodable
+spreaded
```

Comment:
Isn't this a non-flexive verb? https://www.merriam-webster.com/dictionary/spread

Reply:
In ink! we have the concept of a `Spread` storage layout, so "spreaded" refers to having a `Spread`-ed storage layout. I would assume that it's correct usage in this context?

Reply:
Yeah that sounds more correct than "spreaded"